node/core/provisioner/src/lib.rs

```diff
@@ -318,120 +271,20 @@ async fn send_inherent_data(
 	Ok(())
 }
 
-/// In general, we want to pick all the bitfields. However, we have the following constraints:
-///
-/// - not more than one per validator
-/// - each 1 bit must correspond to an occupied core
-///
-/// If we have too many, an arbitrary selection policy is fine. For purposes of maximizing availability,
-/// we pick the one with the greatest number of 1 bits.
-///
-/// Note: This does not enforce any sorting precondition on the output; the ordering there will be unrelated
-/// to the sorting of the input.
-fn select_availability_bitfields(
-	cores: &[CoreState],
-	bitfields: &[SignedAvailabilityBitfield],
-) -> Vec<SignedAvailabilityBitfield> {
-	let mut selected: BTreeMap<ValidatorIndex, SignedAvailabilityBitfield> = BTreeMap::new();
-
-	'a: for bitfield in bitfields.iter().cloned() {
-		if bitfield.payload().0.len() != cores.len() {
-			continue
-		}
-
-		let is_better = selected
-			.get(&bitfield.validator_index())
-			.map_or(true, |b| b.payload().0.count_ones() < bitfield.payload().0.count_ones());
-
-		if !is_better {
-			continue
-		}
-
-		for (idx, _) in cores.iter().enumerate().filter(|v| !v.1.is_occupied()) {
-			// Bit is set for an unoccupied core - invalid
-			if *bitfield.payload().0.get(idx).as_deref().unwrap_or(&false) {
-				continue 'a
-			}
-		}
-
-		let _ = selected.insert(bitfield.validator_index(), bitfield);
-	}
-
-	selected.into_iter().map(|(_, b)| b).collect()
-}
-
 /// Determine which cores are free, and then to the degree possible, pick a candidate appropriate to each free core.
```

Comment:
```suggestion
/// Collect backed candidates with a matching `relay_parent`.
```
We no longer check scheduling or any info about cores